src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<Fruit, Integer> storage = new HashMap<>();
```

Comment:
Storage saves fruits
```suggestion
    public static final Map<Fruit, Integer> fruits = new HashMap<>();
```